ludwig/features/audio_feature.py

```diff
@@ -220,7 +223,7 @@ def _transform_to_feature(
     ):
         feature_type = audio_feature_dict[TYPE]
         if feature_type == "raw":
-            audio_feature = np.expand_dims(audio, axis=-1)
+            audio_feature = np.expand_dims(audio[0], axis=-1)
```

Comment:
Ya good point. So since audio is a less tested feature right now, we definitely need better testing for it. Getting a test together for the the "raw" feature type in addition to a test for the remote capabilities is another (since I've been testing with an internal s3 bucket which seems a bit unorthodox). Since I need to have an audio model trained by Wednesday for the KP presentation however, I kinda need to move quick to get that together before I build out a test suite for the rest of the audio feature pipeline. I should be able to make a lot of progress through the week though assuming the default changes allow me to create the in Predibase.